src/common/injectGlobals.ts

```diff
@@ -28,6 +28,10 @@ export const kfArrangerApiRoot = getApplicationEnvVar('KF_ARRANGER_API');
 export const arrangerProjectId = getApplicationEnvVar('PROJECT_ID');
 export const arrangerApiProjectId = getApplicationEnvVar('PROJECT_API_ID');
 
+export const bannerActivity: boolean = getApplicationEnvVar('BANNER_ACTIVITY') === 'true';
```

Comment:
I would use feature toggle instead, so no need to define this.
Instead of bannerActivity use `isFeatureEnabled('showDynamicBanner')` and in .env the variable will be `REACT_APP_FT_SHOW_DYNAMIC_BANNER=true`

See .env.example for feature toggle usage explanation